docs/data-sources/ipblock.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: "ionoscloud"
+page_title: "IonosCloud: ipblock"
+sidebar_current: "docs-datasource-ipblock"
+description: |-
+Get information on a IonosCloud Ip Block
+---
+
+# ionoscloud\_ipblock
+
+The Ip Block data source can be used to search for and return an existing Ip Block which can
+then be used to provision a server.
+
+## Example Usage
+
+```hcl
+datasource "ionoscloud_ipblock" "example" {
+  id =` <ipblock_id>`
+}
+```
+
+## Argument reference
+
+* `id` - (Optional) ID of an existing Ip Block that you want to search for.
+* `name` - (Optional) Name of an existing Ip Block that you want to search for.
+* `location` - (Optional) ID of the existing Ip Block location.
+
```

Comment:
add here some details about how they should be used. Like either id or name, not both and things like this